package.json

```diff
@@ -16,7 +16,8 @@
     "@nuxtjs/pwa": "^3.1.2",
     "@tailwindcss/ui": "^0.6.2",
     "nuxt": "^2.14.6",
-    "open-color": "^1.8.0"
+    "open-color": "^1.8.0",
+    "ruby": "^0.6.1"
```

Comment:
What does `ruby` do here? It's hard to pick out, but I couldn't spot what was using this.

Reply:
Oh, that's interesting. I wonder what I did to add that. Thanks for pointing this out.